### DISCUSSION TITLE
Problem with sisl.Lattice()

### DISCUSSION THREAD

**Discussion by raulguerrerofis** [zerothi/sisl#612] [General]
Hi Nick,

I am working on a tutorial and I realize that the library sisl.Lattice() is not recognized by sisl.
Are you migrating to another one with another name or something like that?

Best,
Raúl Guerrero

**Comment by zerothi** [comment]
Yes, the next release will be using Lattice instead of SuperCell. So I would suggest you work on the current development, hopefully the next release will be soon out! :)

**Reply by raulguerrerofis** [reply]
Great, thank you :)
